libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController+QrCodeLogin.swift

```diff
@@ -28,44 +28,25 @@
 import Foundation
 
 public extension SalesforceLoginViewController {
-
-    // MARK: - QR Code Login Via UI Bridge API Public Implementation
-
-    /**
-     * Automatically log in with a UI Bridge API login QR code.
-     * - Parameters
-     *     - loginQrCodeContent:  The login QR code content.  This should be either a URL or URL
-     *     query containing the UI Bridge API JSON parameter.  The UI Bridge API JSON parameter
-     *     should contain URL-encoded JSON with two values:
-     *       - frontdoor_bridge_url
-     *       - pkce_code_verifier
-     * If pkce_code_verifier is not specified then the user agent flow is used
-     * - Returns: Boolean true if a log in attempt is possible using the provided QR code content, false
-     * otherwise
-     */
-    func loginFromQrCode(
-        loginQrCodeContent: String?
-    ) -> Bool {
-        if let uiBridgeApiParameters = uiBridgeApiParametersFromLoginQrCodeContent(
-            loginQrCodeContent
-        ) {
-            loginWithFrontdoorBridgeUrl(
-                uiBridgeApiParameters.frontdoorBridgeUrl,
-                pkceCodeVerifier: uiBridgeApiParameters.pkceCodeVerifier
-            )
-            return true
-        } else {
-            return false
-        }
-    }
-
+    
+    // MARK: - QR Code Login Via Salesforce Identity API UI Bridge Public Implementation
+    
     /**
      * Automatically log in with a UI Bridge API front door bridge URL and PKCE code verifier.
+     *
+     * This method is the intended entry point to Salesforce Mobile SDK when using the Salesforce
```

Comment:
I pulled the documentation along to match the Android implementation.  It's very similar reading, so I really tried to match sure I updated all the terminology to iOS SDK context.